images/win/scripts/Installers/Validate-PostgreSQL.ps1

```diff
@@ -0,0 +1,42 @@
+function Stop-PostgreSQL {
```

Comment:
What is the purpose of this function?

Reply:
This function has been created to stop postgresql service after script execution.

Reply:
Why do we need stop one time this service?

Reply:
> Why do we need stop one time this service?

I have implemented this to stop service after all installation+validation steps(like in mysql Ubuntu)

Reply:
This command stopping current psql instance but don't prevent to start the service at windows startup. In that case we should set the service starttype via `Set-Service`.

Reply:
Yes, this part has already been implemented, please check end of script:
```
#Stop and disable PostgreSQL service
Stop-PostgreSQL -PostgresPath $pgroot
Set-Service $pgservice -StartupType Disabled
```

Reply:
Why don't we use built-in Stop-Service cmdlet to stop service and remove function?

Reply:
Agree, thank you. I have removed function and added built-in Stop-Service.